portal/views/admin.py

```diff
@@ -57,7 +58,8 @@
 
 block_limit = 5
 
-recaptcha_client = RecaptchaClient(app_settings.RECAPTCHA_PRIVATE_KEY, app_settings.RECAPTCHA_PUBLIC_KEY)
+recaptcha_client = RecaptchaClient(app_settings.RECAPTCHA_PRIVATE_KEY,
+                                   app_settings.RECAPTCHA_PUBLIC_KEY)
```

Comment:
Majority of changes in this file are PEP-8 related changes to make it vaguely readable.

Actual code changes at `142`, `211`